.github/workflows/assign-pr-number.yml

```diff
@@ -46,7 +46,8 @@ jobs:
         git config user.name "$author_name"
         if [ -n "$(git diff --staged)" ]; then
           git commit \
-            -m "docs: Rename the news fragment with the PR number\n\n${{ join(fromJSON(steps.assign_pr_number.outputs.rename_results), '\n') }}" \
+            -m "docs: Rename the news fragment with the PR number" \
+            -m "${{ join(fromJSON(steps.assign_pr_number.outputs.rename_results), '\n') }}" \
```

Comment:
This seems to make "\n" to be present as the literal, not newlines.

`git commit -m 'xxx: test' -m "yyy\nzzz\nqqq"`
→
<img width="301" alt="image" src="https://github.com/user-attachments/assets/67f4a7b5-a415-49dc-bfbc-aed5f7a94dd0" />

Let's try:
git commit -m 'xxx: test' -m $'yyy\nzzz\nqqq'`
→
<img width="306" alt="image" src="https://github.com/user-attachments/assets/46fd1fc6-823b-47c5-887f-5f9a18115a18" />